### PR TITLE
fix save checkpoint error with resume training at pytorch 1.6

### PR DESCRIPTION
**Pull request by halostorm**
Please refer to https://github.com/open-mmlab/mmsegmentation/issues/127 for details.

### CODE REVIEW

**Comment by hellock** [comment]
ping @yhcao6

File: mmcv/runner/hooks/optimizer.py [hunk 1]
```diff
@@ -63,12 +63,11 @@ def __init__(self,
     def before_run(self, runner):
         """Preparing steps before Mixed Precision Training.
 
-        1. Make a master copy of fp32 weights for optimization.
+        1. Deepcopy model to segregate model's parameters from optimizer.
```
Comment (yhcao6) [inline mmcv/runner/hooks/optimizer.py:66]: Specify the precision of weights of model and optimizer respectively.

**Comment by lartpang** [comment]
This BUG is actually very important...
Thanks!

**Comment by yhcao6** [comment]
Dear @halostorm, thanks for your contribution, we compare this Pr with #580 and found #580 is more memory friendly, so we adopt that scheme. Feel free to reopen this pr if you have any questions.

**Pull request closed without merging**